test/unit/gateways/cyber_source_test.rb

```diff
@@ -849,10 +849,33 @@ def test_adds_application_id_as_partner_solution_id
     end.check_request do |endpoint, data, headers|
       assert_match("<partnerSolutionID>#{partner_id}</partnerSolutionID>", data)
     end.respond_with(successful_capture_response)
-  ensure
```

Comment:
Was this `ensure` line intended to be removed?

Reply:
Nope. Must have read it as an artifact of my "end" fix. I dug up the original version of this function and copy/pasted it back to make sure it is exactly like it was before.

https://github.com/activemerchant/active_merchant/blob/277b123d1f3da81464a6515c355635923d6a4805/test/unit/gateways/cyber_source_test.rb#L852

Reply:
Looks good! Thanks for making those fixes!